### PR TITLE
implement athb & athb_array, test and index included

### PR DESCRIPTION
**Pull request by Lu-Whale**
function athb & athb_array in athb.js

### CODE REVIEW

File: src/models/athb.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
+/**
+ * Return the PMV value calculated with the Adaptive Thermal Heat Balance
+ * Framework [27]_. The adaptive thermal heat balance (ATHB) framework
+ * introduced a method to account for the three adaptive principals, namely
+ * physiological, behavioral, and psychological adaptation, individually
+ * within existing heat balance models. The objective is a predictive model of
+ * thermal sensation applicable during the design stage or in international
+ * standards without knowing characteristics of future occupants.
+ *
```
Comment (szaldivar) [inline src/models/athb.js:9]: add `@public` so it gets included in the documentatation

File: src/models/athb.js [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+ * // Import the athb function from the jsthermalcomfort module
+ * import { athb } from "../../src/models/athb.js";
+ *
+ * // Calculate PMV with sample data when it is scalar arguments.
+ * console.log(athb({ tdb: 25, tr: 25, vr: 0.1, rh: 50, met: 1.1, t_running_mean: 20 }));
+ * // Output: 0.2
+ *
+ */
+
```
Comment (szaldivar) [inline src/models/athb.js:44]: remove empty line

File: src/models/athb.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+ * @returns { number } athb_pmv - Predicted Mean Vote calculated with the Adaptive Thermal Heat Balance framework
+ *
+ * @example
+ * // JavaScript code example
+ *
+ * // Import the athb function from the jsthermalcomfort module
+ * import { athb } from "../../src/models/athb.js";
+ *
+ * // Calculate PMV with sample data when it is scalar arguments.
+ * console.log(athb({ tdb: 25, tr: 25, vr: 0.1, rh: 50, met: 1.1, t_running_mean: 20 }));
```
Comment (szaldivar) [inline src/models/athb.js:40]: this example is wrong, in the function you take params but here you are passing an object

File: src/models/athb.js [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+ *
+ * // Import the athb_array function from the jsthermalcomfort module
+ * import { athb_array } from "../../src/models/athb.js";
+ *
+ * // Calculate PMV with sample data when it is arrays.
+ * console.log(athb({ tdb: [25, 27], tr: [25, 25], vr: [0.1, 0.1], rh: [50, 50], met: [1.1, 1.1], t_running_mean: [20, 20] }));
+ * // Output: [0.2, 0.209]
+ */
+export function athb_array(tdb, tr, vr, rh, met, t_running_mean) {
+  tdb = Array.isArray(tdb) ? tdb : [tdb];
```
Comment (szaldivar) [inline src/models/athb.js:124]: no point in doing these checks, your params are arrays

File: src/models/athb.js [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+        0.0342 * t_running_mean[index] +
+        0.0002264 * lValue * t_running_mean[index] +
+        0.018696 * met_adapted[index] * t_running_mean[index] -
+        0.0002909 * lValue * met_adapted[index] * t_running_mean[index]
+      ).toFixed(3),
+    ),
+  );
+}
+
+function pmv_ppd_optimized(tdb, tr, vr, rh, met, clo, wme) {
```
Comment (szaldivar) [inline src/models/athb.js:181]: why is this function here? (should not include double implementations in multiple PR)
also why is it called optimized?
what is optimized about it? (aka this does not make sense in the javascript version) (the python version was using a vectorized technique, also using JIT and caching) (here we are not doing any of that so no point in calling it optimized)

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: src/models/athb.js (hunk 5 above)
Comment (Lu-Whale) [reply src/models/athb.js:181]: so here, should I use the function pmv instead?

**Pull request closed without merging**